leipzig-gloss.typ

```diff
@@ -40,79 +40,112 @@
 #let cmdlabel = label 
 
 #let gloss(
-    header: none,
-    header-style: none,
-    source: (),
-    source-style: none,
-    transliteration: none,
-    transliteration-style: none,
-    morphemes: none,
-    morphemes-style: none,
-    additional-lines: (), //List of list of content
-    translation: none,
-    translation-style: none,
     label: none,
     label-supplement: [example],
-
-    item-spacing: 1em,
     gloss-padding: 2.0em, //TODO document these
     left-padding: 0.5em,
     numbering: false,
     breakable: false,
+    ..args
 ) = {
+    // Typesets the internal part of the interlinear glosses. This function does not deal with the external matters of numbering and labelling.
+    let typeset_gloss(
```

Comment:
Typst style is apparently to use `kebab-case` not `snake_case`, and the package submission system now lints for that.